backlog_controller.hh

```diff
@@ -78,7 +78,7 @@ protected:
 
     void adjust();
 
-    backlog_controller(scheduling_group& sg, std::chrono::milliseconds interval,
+    backlog_controller(const scheduling_group& sg, std::chrono::milliseconds interval,
```

Comment:
nit: should probably be just a value at this point

Reply:
Yeah, it should work

Reply:
done in v6